### PR TITLE
feat: set greyscale icon when not found Web Components

### PR DESCRIPTION
**Pull request by jorgecasar**

### CODE REVIEW

File: src/background_script.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const metaAssets = '../meta_assets/';
+var selectedId = -1;
+
+function getIconPath(response) {
+	const sufix = response.length ? '' : `_without`;
```
Comment (thepassle) [inline src/background_script.js]: Im getting an error in chrome on this line:

`Error handling response: TypeError: Cannot read property 'length' of undefined`

image

File: src/content_script.js [hunk 2]
```diff
@@ -16,20 +16,33 @@ function findAllCustomElements(nodes) {
...
-  if(request.msg === "init") {
+
+  if(request.msg === "findAll") {
     allCustomElements = [];
     findAllCustomElements(document.querySelectorAll('*'));
     sendResponse(allCustomElements)
   }
 
+  if(request.msg === "init") {
+    // They are already finded.
```
Comment (thepassle) [inline src/content_script.js]: Can we change this to: `// Custom elements are already found`? 🙂

**Comment by thepassle** [comment]
Could you check if your branch is completely up to date? There seems to be some code in your branch that shouldve been deleted already 🤔 https://github.com/jorgecasar/locator/blob/enable-icon-on-found/src/found-element.js#L33

File: src/background_script.js (hunk 1 above)
Comment (thepassle) [reply src/background_script.js]: also please rename to `suffix` instead of `sufix` 😄

Comment (jorgecasar) [reply src/background_script.js]: Fixed, sometimes the responseCallback provide an undefined. I set a default empty array.

File: src/content_script.js (hunk 2 above)
Comment (jorgecasar) [reply src/content_script.js]: Done!

**Comment by jorgecasar** [comment]
> Could you check if your branch is completely up to date? There seems to be some code in your branch that shouldve been deleted already 🤔 https://github.com/jorgecasar/locator/blob/enable-icon-on-found/src/found-element.js#L33

Reabased!

**Comment by thepassle** [comment]
LGTM, could you update the version in the `manifest.json` to `1.1` and in the package.json to `1.1.0`? 🙂

Thanks for the contribution! Very fun feature :)

**Comment by jorgecasar** [comment]
Sure! 

You are welcome!

**Comment by thepassle** [comment]
Thanks! It'll take a couple of days before it gets released because the chrome extension stuff is very slow, but this will be available on the next release 🙂

**Pull request merged**